commit 972b68069c03cae7f92e2637ff205b8e1dc1f9dd
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-07-09T15:09:53-04:00 (3 months ago)
    
    TRITON-1804 booter's node_module vendoring breaks if "build" directory does not exist

diff --git a/Makefile b/Makefile
index efdf2a2..db62eaf 100644
--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,7 @@ TOP := $(shell pwd)
 #
 TAPE := ./node_modules/.bin/tape
 ISTANBUL := ./node_modules/.bin/istanbul
-PACK := ./build/pack-0.0.1.tgz
+PACK := ./$(BUILD)/pack-0.0.1.tgz
 
 #
 # Files
@@ -95,7 +95,8 @@ coverage: | $(ISTANBUL) $(TAPE) node_modules
 
 $(PACK):
 	$(NPM) pack file:$(TOP)/src/node-pack
-	mv pack-0.0.1.tgz build/
+	mkdir -p $(TOP)/$(BUILD)
+	mv pack-0.0.1.tgz $(TOP)/$(BUILD)/
 
 #
 # Packaging targets
